### ISSUE TITLE
Getting Unavailable on all sensors

### ISSUE THREAD

**Issue by ncmarket** [briis/weatherflow_forecast#150]
**Describe the bug**
Since the Home Assistant 2024.08 release, I have been getting on all of my WeatherFlow devices and sensors "Unavailable" . 

Using WeatherFlow swagger, I confirmed that the API key was valid and showed the observation data for the Tempest device.

I deleted the WeatherFlow API key and built a new key using the WeatherFlow methods. This restored the measures, but it failed after 12 hours.

**To Reproduce**
Check Developer Tools and States for station measurements. Or check dashboards containing WeatherFlow measurements, they will show as Unavailable.

**Expected behavior**
I expected sensor data to be available and usable.

**Screenshots**
 
<img width="515" alt="Screenshot 2024-08-12 at 10 31 01 AM" src="https://github.com/user-attachments/assets/b9ac15e7-0ec9-4ca6-a8d3-818b161015d5">

**Additional context**
No additional context

**Comment by briis** [comment]
I cannot reproduce this. Everything is working in my two installations.
Could you have a look in the Log to see if anything shows up here, and post that?

If nothing, try enable debug logging like described here: https://github.com/briis/weatherflow_forecast?tab=readme-ov-file#enable-debug-logging

**Comment by ncmarket** [comment]
Nothing in the logs. I will enable debug logging.

**Comment by ncmarket** [comment]
After adding the section described in the link above to configuration.yaml and reloading the yaml,  from weatherflow-forecast nothing appeared on the logs. I restarted Home Assistant, went to the integration to check the configuration, and found that I could not authenticate to Weatherflow - red tint box stating authentication error.

I was unable to check the configuration using the UI, so I deleted the integration, restarted Home Assistant, and reinstalled it. The installation went fine; I provided the correct station ID and the Weatherflow API key. The integration has been running for the past 48 hours.

I have resolved my issue.
